src/lib/offline-api/validator/errors.ts

```diff
@@ -84,6 +84,11 @@ const errors = {
       }
     }
   },
+  editorLayout: {
+    FIELD_GROUP_LEVEL_TOO_DEEP: () => {
+      return 'You cannot create a field group deeper that 2 levels'
```

Comment:
nit: impersonal messages are used for all other errors but TAGS_DO_NOT_EXIST

Reply:
updated